view/theme/frio/js/theme.js

```diff
@@ -764,12 +764,49 @@ function htmlToText(htmlString) {
  * @param {boolean} un    Whether to perform an activity removal instead of creation
  */
 function doActivityItemAction(ident, verb, un) {
-	if (verb.indexOf("attend") === 0) {
-		$(".item-" + ident + " .button-event:not(#" + verb + "-" + ident + ")").removeClass("active");
-	}
-	$("#" + verb + "-" + ident).toggleClass("active");
-
-	doActivityItem(ident, verb, un);
+	_verb = un ? 'un' + verb : verb;
+	$('#like-rotator-' + ident.toString()).show();
+	$.post('item/' + ident.toString() + '/activity/' + _verb)
+	.success(
+		function(data){
+			if (data.status == "ok") {
```

Comment:
Please use single quotes when double-quotes aren't warranted.
```suggestion
			if (data.status == 'ok') {
```